commons/src/main/scala/za/co/absa/spline/common/validation/ValidationUtils.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.common.validation
+
+import jakarta.validation.ConstraintViolationException
+
+object ValidationUtils {
+
+  @throws[ConstraintViolationException]
+  def validate(c: Constraint): Unit = {
+    validate(c.isValid, c.message)
+  }
+
+  @throws[ConstraintViolationException]
+  def validate(b: Boolean, msg: => String): Unit = {
+    if (!b) throw new ConstraintViolationException(msg, null) // NOSONAR
+  }
```

Comment:
- I don't understand why is this separated method. Could both validate methods be just merged into one?
- the name doesn't describe what the method does. Maybe `throwIfNotValid` would make more sense

Reply:
It's a utility object that aims for the samilar purpose as JSR-380, but in a simplistic implementation without annotations and external dependencies. The method is called `validate()` as the standard `Validator.validate()`, and it's overloaded to support simple checks that don't necessarily require a dedicated constraint class. E.g.
```scala
ValidationUtils.validate(age > 0, "Invalid age: $age")
```

Reply:
ok, used like that it makes more sense